wasp/widgets.py

```diff
@@ -285,3 +285,80 @@ def touch(self, event):
         elif v >= self._steps:
             v = self._steps - 1
         self.value = v
+
+
+_message_string_x_coord = const(0)
+_message_string_y_coord = const(60)
+_yes_button_x_coord = const(20)
+_yes_button_y_coord = const(100)
+_no_button_x_coord = const(120)
+_no_button_y_coord = const(100)
+
+class ConfirmationView:
+    "Confirmation widget allowing user confirmation of a setting"
+
+    class TouchResult:
+        YES = 0
+        NO = 1
+        NO_ACTION = 2
+
+    def __init__(self):
+        self.drawable = wasp.watch.drawable
```

Comment:
Don't cache the drawable in the instance. It does not have the same effect as caching within a single function.

Reply:
:+1: cool thanks have changed